frontend/src/components/Table/Table.scss

```diff
@@ -1,16 +1,26 @@
+$table-color: #dee2e6;
```

Comment:
/nit: I'd love to see this moved to one of our "global" SCSS files - like  `colors.scss`